lua/exporters/object.lua

```diff
@@ -68,4 +68,24 @@ end
 
 GRPC.exporters.object = function(object)
   return {}
-end
\ No newline at end of file
+end
+
+GRPC.exporters.markPanel = function(markPanel)
+  local mp = {
+    id = markPanel.idx, 
+    time = markPanel.time,
+    initiator = GRPC.exporters.unit(markPanel.initiator),
+    text = markPanel.text,
+    position = toLatLonPosition(markPanel.pos)

```

Comment:
Just a minor thing, because I'll probably not remember the line when merging my PR in some weeks. Would you mind changing this to the following already?

```suggestion
    position = GRPC.toLatLonPosition(markPanel.pos)
```

Reply:
Done